src/electron/electron/context_menu.cljs

```diff
@@ -0,0 +1,95 @@
+(ns electron.context-menu
+  (:require [clojure.string :as string]
+            [electron.utils :as utils]
+            ["electron" :refer [Menu MenuItem shell] :as electron]
+            ["electron-dl" :refer [download]]))
+
+;; context menu is registerd in window/setup-window-listeners!
+(defn setup-context-menu!
+  [^js win]
+  (let [web-contents (. win -webContents)
+
+        context-menu-handler
+        (fn [_event params]
```

Comment:
`params` here can be annotated with `^js` so that other params below don't need to be annotated again.